src/app/components/ScrollablePromo/index.jsx

```diff
@@ -41,21 +61,57 @@ const ScrollablePromo = ({ blocks, blockGroupIndex }) => {
     return null;
   }
 
-  const blocksWithoutTitle = blocks[0].type === 'title' ? tail(blocks) : blocks;
+  const blocksWithoutTitle =
+    blocks[0]?.type === 'title' ? tail(blocks) : blocks;
 
   const isSingleItem = blocksWithoutTitle.length === 1;
 
+  const title = pathOr(
+    'You may also be interested in',
+    ['recommendationTitle'],
+    translations,
+  );
+
+  const labelId = 'recommendations-heading';
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': labelId,
+  };
+
   return (
-    <GridItemMediumNoMargin>
+    <GridItemMediumNoMargin
+      {...(recommendations && { 'data-e2e': { labelId }, ...a11yAttributes })}
+    >
+      {recommendations && (
+        <LabelComponent
+          script={script}
+          service={service}
+          dir={dir}
+          labelId={labelId}
+          columnType="main"
+          overrideHeadingAs="strong"
+          bar={false}
+          backgroundColor={C_GHOST}
```

Comment:
It looks like this background colour isn't applying. It might be best to move it to the styled component on line 41, instead of a prop on the component?

Reply:
Do we have the design for this yet? Think we will also need to change the typography settings, looks a little large...

Reply:
Not sure I would use section label here... Is that what we use for the WSOJ component?

Reply:
@HarveyPeachey What designs are you looking at? I don't have the latest designs yet, I was just planning ahead and wanted some early review on the ticket. For now, I followed the design we use on the CPS Recommendation.

Reply:
@greenc05 No, I don't have the design yet, I am currently following the title/section label design from the control version of the experiment. I will update them once I have the design in my hands. I was just planning ahead and wanted some early review on the way I am making the EOJ component accept recommendation data. Maybe I should have pointed this out somewhere 😅.

Reply:
I got confused, as one of the designs in the linked zeplin had `hover` above the scrollable link, which looked like a title. But I can't see the new designs, as it says I don't have access to the workspace? I'll resolve as I assume you've mirrored the designs